Services/UI/OJS.Services.Ui.Business/Validation/ValidationMessages.cs

```diff
@@ -16,6 +16,8 @@ public static class Contest
     public static class User
     {
         public const string NotLoggedIn = "You need to log in first.";
+
+        public const string NotAdminOrLecturer = "You do not have permissions to download the file.";
```

Comment:
Is is possible to make this a generic error message so it can be used no matter what the user is unauthorized about? Maybe leave it to "You do not have permissions for this action."?

Reply:
Done.